Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -32,6 +32,21 @@ spec:
       dataChunks: 2
       codingChunks: 1
   preservePoolsOnDelete: true
+  auth:
+  #  keystone:
+  #    acceptedRoles:
+  #      - admin
+  #      - member
+  #      - service
+  #    implicitTenants: "swift"
+  #    revocationInterval: 1200
+  #    serviceUserSecretName: usersecret
+  #   tokenCacheSize: 1000
+  #    url: https://keystone.example-namespace.svc/
+  protocols:
+    swift:
```

Comment:
Unless this is somehow required, can we have this commented-out by default?

Reply:
83aacf43934beb55209fe908c43dff7b2af83661

(Fixups will be squashed if review suggestions are coming to an end.)